wolox-books/src/app/screens/log-in/log-in.component.scss

```diff
@@ -0,0 +1,7 @@
+@import './src/scss/colors';
+
+:host {
```

Comment:
if this repeats in both login and sign up add a class to each container and move this styles to the generic styles file